cmd/snap-confine/mount-support.c

```diff
@@ -648,7 +648,6 @@ void sc_populate_mount_ns(struct sc_apparmor *apparmor, int snap_update_ns_fd,
 			{"/sys"},	// fundamental filesystem
 			{"/tmp"},	// to get writable tmp
 			{"/var/snap"},	// to get access to global snap data
-			{"/var/lib/dhcp",.is_optional = true}, // to support network-control interface
```

Comment:
Why not (at least) also fix /var/log (log-observe, account-control, ppp), /usr/src (system-trace), /mnt (removable-media), /var/lib/extrausers (default template, account-control, classic-support). I left /media (/run/media) since it is #ifdef'd, but it could also be moved to removable-media.

Reply:
I think for those I'd like to iterate and tackle the permission problem, so that it is easy to express the permissions without having complex review required.